pkg/controller/workload/workload_event.go

```diff
@@ -108,19 +108,6 @@ func (svc *ServiceEvent) processWorkloadResponse(rsp *service_discovery_v3.Delta
 			err = fmt.Errorf("unsupport type url %s", rsp.GetTypeUrl())
 		}
 	}
-
```

Comment:
is there a situation we need to consider？  rsp.GetResources() == nil & rsp.RemovedResources != nil